doc/tutorial/tuto_1_1.py

```diff
@@ -18,7 +18,7 @@ def module_to_dot(dependencies, targets):
     for dep in dependencies:
         filepath = pathlib.Path(dep)
         source = filepath.stem
-        with open(filepath) as fh:
```

Comment:
What is the error you were getting? `open` is a valid python built-in function...

Reply:
Yes it is, however it only supports `pathlib.Path` objects since python 3.6. I was using python 3.5.2 where this raises a `TypeError`. From your AppVeyor setup I see that you plan to support python3.4 and 3.5 as well so you'd need this for compatibility.

I've used `open` with `Path` objects myself and always got that `TypeError`, at least that's what I thought.
Until this morning, when I found out that it actually _does_ work with >=3.6. That was very confusing for me since I was certain that I've tried with 3.6 and 3.7 already. Anyway, so its not a bug but a compatibility issue.

Reply:
ah. ok... it is bug.